src/chapters/aspdotnet-model-validation/studio.rst

```diff
@@ -0,0 +1,120 @@
+Studio: Spa User Validation
+===========================
+
+We’ll build on the :ref:`User Signup <user-signup-studio>` studio from last
+class, adding in model validation.
+
+Getting Started
+---------------
+
+Open up your ``SpaDay`` application and checkout the `user-signup-pt2 <https://github.com/LaunchCodeEducation/SpaDay/tree/user-signup-pt2>`__ branch. 
+
+Creating a New ViewModel
+------------------------
+
+To get started with adding validation to our application, we need to first make a ViewModel.
+After adding a folder for storing our ViewModels, create a new ViewModel for working with form submission called ``AddUserViewModel``.
+Add properties for the user's username, password, and email.
+Also, add a new property called ``VerifyPassword``.
+
+Add Validation Attributes
+-------------------------
+
+Navigate to the ``AddUserViewModel`` model class. Add `validation
```

Comment:
id assume from the last instruction we are still in the viewmodel class

Reply:
Done!